source/base/subscriptor.cc

```diff
@@ -101,7 +101,7 @@ void Subscriptor::check_no_subscribers () const
             infostring = "<none>";
 
           AssertNothrow (counter == 0,
-                         ExcInUse (counter, object_info->name(), infostring));
+                         ExcInUse (counter.load(), object_info->name(), infostring));
```

Comment:
Any reason for changing this? I am fine with both.

Reply:
I don't think it matters if I include the load or not, but I have a slight preference for being explicit in how we use atomic values. I omitted the first one since its more confusing.